IAGrim/UI/Tabs/ModsDatabaseConfig.cs

```diff
@@ -140,6 +141,12 @@ public void ForceDatabaseUpdate(string location, string modLocation)
                 var mod = listViewMods.SelectedItems[0].Tag as ListViewEntry;
                 var entry = lvi.Tag as ListViewEntry;
 
+                if (mod != null)
+                {
```

Comment:
Really looks like it ignored editorconfig on this one, but no biggie.

Reply:
Fixed.